### DISCUSSION TITLE
右手キーが反応しない

### DISCUSSION THREAD

**Discussion by KazukiNagao** [Yowkees/keyball#420] [Q&A] [answered]
Keyball44の右手キーボードにていくつか反応しないキーがあります。
どこを見直せばよいのか分からず、教えていただきたいです。

添付画像赤枠部分のキーが反応しないです。
![IMG_2213](https://github.com/Yowkees/keyball/assets/5966602/e824d5c6-5e1b-49c6-a6f4-9f77e64180ca)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

反応しないキーがバラバラの位置にありますので、おそらく反応しないキーのキーソケットもしくは近くのダイオードの半田付け不良が原因と思われます。
キーソケットの半田付けはとてもテンプラしやすいので、これを１つずつご確認ください。

キーソケットの半田が多いように見えますので、半田吸い取り線等で量を減らしていただき、下写真の緑矢印部分の端子と基板の間にある隙間に注目して半田を足してください。ここから半田が染み出してくるのが見えれば、テンプラ無く確実に半田付け出来ています。

お手数ですが半田付け状況のご確認お願いいたします。

![kb44_080](https://github.com/Yowkees/keyball/assets/84209772/add17b54-8fac-4c44-bf7a-ede8859d317c)

**Reply by curuknagao** [reply]
キーソケットのハンダ見直しまして、以下の２点は動作するようになりましたが、他の部分はどうにも動かないんですが、対応するダイオードはどこになるか教えて頂くことは可能でしょうか？
![282250620-e824d5c6-5e1b-49c6-a6f4-9f77e64180ca](https://github.com/Yowkees/keyball/assets/30860279/84e72c94-8b92-49d0-8aff-c6386cf29dc6)

**Reply by Ryo624** [reply]
キーソケットのはんだ付けの見直しを実施していただきましてありがとうございます。

改善したキー以外のキーに対応するダイオードは写真にあります色どうしのものになります。
お手数ですが、もう一度はんだの見直しをお願いいたします。

![282506258-84e72c94-8b92-49d0-8aff-c6386cf29dc6](https://github.com/Yowkees/keyball/assets/143698349/cee15318-c672-4e95-882d-d5870dca56c4)

**Reply by KazukiNagao** [reply, marked as answer]
こちらpromicroのハンダ不足が原因でした。
ありがとうございました。
